src/ontology/mondo-ingest.Makefile

```diff
@@ -112,8 +176,8 @@ $(COMPONENTSDIR)/ncit.owl: $(TMPDIR)/ncit_relevant_signature.txt | component-dow
 		annotate --ontology-iri $(URIBASE)/mondo/sources/ncit.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/ncit.owl -o $@; fi
 
 # todo: See #1 at top of file
-$(COMPONENTSDIR)/doid.owl: $(TMPDIR)/doid_relevant_signature.txt | component-download-doid.owl
-	if [ $(COMP) = true ]; then $(ROBOT) remove -i $(TMPDIR)/component-download-doid.owl.owl --select imports \
+$(COMPONENTSDIR)/doid.owl: $(TMPDIR)/doid_relevant_signature.txt | $(TMPDIR)/mirror-doid.owl
+	if [ $(COMP) = true ]; then $(ROBOT) remove -i $(TMPDIR)/mirror-doid.owl --select imports \
```

Comment:
@matentzn when I run `sh run.sh make components/doid.owl -B`, the file has this:

`<owl:versionIRI rdf:resource="http://purl.obolibrary.org/obo/mondo/sources/2025-07-21/doid.owl"/>` - this is the date I ran the make command

and then these:
```
<owl:versionInfo>2025-06-27</owl:versionInfo>
<owl:versionInfo>2025-07-21</owl:versionInfo>
```

Is this what is intended?

Reply:
THANK YOU! If you could. thanks

Reply:
new feature branch data build https://github.com/monarch-initiative/mondo-ingest/pull/856

Reply:
Seems I wrote a long answer to this and it got lost.

Short version now: The missing ones need to be added outside of Mondo ingest:

- icd10who: https://github.com/monarch-initiative/icd10who/issues/5
- omim: https://github.com/monarch-initiative/omim/issues/240
- icd11: https://github.com/monarch-initiative/icd11/issues/19

Reply:
We also dont need to wait for this PR to be merged, we can merge and deal with the others in parallel